scripts/qa_crowbarsetup.sh

```diff
@@ -4922,6 +4922,9 @@ zypper -non-interactive --gpg-auto-import-keys --no-gpg-checks install ses-upgra
     for node in $ceph_mons; do
         ssh $node "ceph osd crush tunables firefly; ceph osd set require_jewel_osds"
     done
+
+    # wait for ceph cluser to recover after the upgrade
+    sleep 60
```

Comment:
nope, need to grep for the status because with `HEALTH_WARN` the exit code is still 0:

```
root@d52-54-77-77-01-01:~ # ceph status
    cluster c4cf7412-a4a4-4122-a8d2-c4c8091d3e4c
     health HEALTH_WARN
            192 pgs degraded
            66 pgs stuck unclean
            192 pgs undersized
            recovery 58/174 objects degraded (33.333%)
            1/3 in osds are down
     monmap e1: 1 mons at {d52-54-77-77-01-01=192.168.253.83:6789/0}
            election epoch 4, quorum 0 d52-54-77-77-01-01
     osdmap e43: 3 osds: 2 up, 3 in; 192 remapped pgs
            flags require_jewel_osds
      pgmap v1803: 192 pgs, 3 pools, 165 MB data, 58 objects
            583 MB used, 52631 MB / 53214 MB avail
            58/174 objects degraded (33.333%)
                 192 active+undersized+degraded
root@d52-54-77-77-01-01:~ # echo $?
0
```